packages/svelte-ux/src/lib/components/ThemeSwitch.svelte

```diff
@@ -4,9 +4,18 @@
   import Switch from './Switch.svelte';
   import Icon from './Icon.svelte';
 
-  import { getSettings } from './settings.js';
+  import { getComponentSettings, getSettings } from './settings.js';
+  import { cls } from '../utils/index.js';
+  import type { ComponentProps } from 'svelte';
+  import { clsMerge } from '$lib/utils/styles.js';
 
   const { currentTheme } = getSettings();
+
+  const { classes: settingsClasses } = getComponentSettings('ThemeSwitch');
```

Comment:
Not sure if there's a meaningful difference between `getComponentSettings` and `getComponentClasses` in this case

Reply:
Awe, good point.  `getComponentSettings` is only useful when there are other settings/defaults to read (`variant`,  `labelPlacement`, etc).

With that said, feel free to use `const settingsClasses = getComponentClasses('ThemeSwitch');` to streamline this a bit.